src/components/forms/formr-part-a/View.cy-test.tsx

```diff
@@ -53,10 +53,16 @@ describe("View", () => {
     cy.get("[data-cy=BtnEdit]").should("not.exist");
     cy.get("[data-cy=BtnSaveDraft]").should("not.exist");
     cy.get("[data-cy=BtnSubmit]").should("not.exist");
-    cy.get("[data-cy=submissionDateTop]").should("exist");
```

Comment:
Sorry to be picky, Ed but I think it's better to keep the `data-cy=submissionDateTop` tags etc. you had earlier rather than `#__cy_root > :nth-child(1) > :nth-child...` to keep it isolated from any changes we might make - and it's also easier to read!